ccf_transaction_processor/README.md

```diff
@@ -24,9 +24,12 @@ TP. Make sure the following env variables are defined:
 who location is pointed to by this env variable. Note that this is the same key that will be
 used to sign the pdo contract enclaves.
 
-5) `source ${PDO_SOURCE_ROOT}/build/common-config.sh` to define some of the dependent env variables 
+5) `source ${PDO_SOURCE_ROOT}/build/common-config.sh` to define some of the dependent env variables
 (such as PDO_HOME)
 
+IMPORTANT: When installing CCF and PDO on the same VM for local testing, please install PDO first and
+then CCF. See [PDO docs](../docs) for detailed instructions on installing PDO.
+
 ## Get CCF Source Code
```

Comment:
can we just suggest that CCF and PDO are installed on separate servers?

Reply:
Everything works just fine with both in the same node. We may not want to underplay it.

Reply:
Again... i'm ok with leaving it as is, but i think we have already spent too much effort (and will continue to make effort as we maintain this) making this work when it is not necessary.